src/components/elements/Inputs/MyAccountDropdown/MyAccountDropdown.tsx

```diff
@@ -74,7 +75,9 @@ const MyAccountDropdown = (props: PropsWithChildren<MyAccountDropdownProps>) =>
       }
     } else if (item.value === "Logout") {
       removeAccessToken();
-      router.push("/auth/login");
+      router.push("/auth/login").then(() => {
+        ApiSlice.clearApiCache();
+      });
```

Comment:
This isn't sufficient. Please use the `logout` function from the v3 utils: https://github.com/wri/wri-terramatch-website/blob/41d485d32cf95655f86f7839362fb42beff68db6/src/generated/v3/utils.ts#L138